nuxt.config.ts

```diff
@@ -69,7 +69,7 @@ export default defineNuxtConfig({
       alias: {
         process: 'process/browser',
         stream: 'stream-browserify',
-        https: 'agent-base',
+        https: 'stream-http',
```

Comment:
What does this mean? There is another lib that needs to be used now? (I'm not super familiar with Nuxt config - it caught my curiosity)

Reply:
I was just wrong one we used before. But yes this is for Nuxt